setup.py

```diff
@@ -22,13 +22,13 @@
     version += ".post" + str(int(time.time()))
 
 install_requires = [
-    "numpy>=1.14",
+    "numpy<1.19.0,>=1.16.0",
     "pandas>1.0.3,<2",
-    "scipy>=1.0.0,<2",
+    "scipy==1.4.1",
```

Comment:
also, not sure if pinning scipy can generate problems, Abir, does the code break with `scipy>=1.0.0,<2`